acceptance/current/.acceptance/acceptance-cookbook/recipes/destroy.rb

```diff
@@ -0,0 +1,3 @@
+execute 'kitchen destroy' do
+  cwd File.join(File.dirname(__FILE__), "../../../../../../../..")
```

Comment:
Ugh this feels wrong. Perhaps chef-acceptance can inject some node attributes into the chef-zero run? Something like `node['chef-acceptance']['root']` or `node['chef-acceptance']['suite_root']`.